src/main/resources/i18n/OrionBundle.properties

```diff
@@ -25,7 +25,7 @@ orion.dialog.submissionDeletion.label=Your currently open submission will be del
 orion.dialog.feedbackOverwrite.title=Overwrite Feedback?
 orion.dialog.feedbackOverwrite.label=Artemis has been reloaded. Do you want to replace the current local feedback with the feedback stored on the server?
 orion.vcs.updating=Updating your exercise files...
-orion.vcs.multipleremotes=Multiple remote branches detected. Your changes might got pushed to a branch that will not be assessed.
+orion.vcs.multipleremotes=Multiple remote branches detected. Your changes might get pushed to a branch that will not be assessed.
```

Comment:
Im not quite sure, but I would assume the message is shown after the push - in that case it needs to be past tense

Reply:
Looking at it that way, we have to change the whole sentence. 
`got pushed` is not valid because the verb might is a modal verb.
I read a bit further into this article and think the form might get pushed is correct.
https://www.grammarly.com/blog/may-might/